### PR TITLE
feat: repository encryption parameter

### PR DESCRIPTION
**Pull request by brivu**
This `PR` adds the `--encryption-configuration` parameter to the `aws ecr create-repository` command, enabling users to set the ECR repo encryption while the repository is being set up.

### CODE REVIEW

File: src/scripts/create_repo.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ORB_STR_REGION="$(circleci env subst "${ORB_STR_REGION}")"
 ORB_STR_REPO="$(circleci env subst "${ORB_STR_REPO}")"
 ORB_STR_PROFILE_NAME="$(circleci env subst "${ORB_STR_PROFILE_NAME}")"
+ORB_ENUM_ENCRYPTION_TYPE="$(circleci env subst "${ORB_ENUM_ENCRYPTION_TYPE}")"
+ORB_STR_ENCRYPTION_KMS_KEY="$(circleci env subst "${ORB_STR_ENCRYPTION_KMS_KEY}")"
```
Comment (fruit) [inline src/scripts/create_repo.sh:6]: This 2 lines are duplicated - already exists on 12-13 lines

File: src/scripts/create_repo.sh [hunk 2]
```diff
@@ -2,15 +2,34 @@
...
```
Comment (fruit) [inline src/scripts/create_repo.sh:6]: Given 2 lines are exists on 12-13 lines. Remove them here, or remove them from 12-13 lines.

```suggestion
```

File: src/scripts/create_repo.sh [hunk 3]
```diff
@@ -2,15 +2,32 @@
...
 else
-    aws ecr describe-repositories --profile "${ORB_STR_PROFILE_NAME}" --region "${ORB_STR_REGION}" --repository-names "${ORB_STR_REPO}" >/dev/null 2>&1 ||
-        if [ "$ORB_BOOL_REPO_SCAN_ON_PUSH" == "1" ]; then
-            aws ecr create-repository --profile "${ORB_STR_PROFILE_NAME}" --region "${ORB_STR_REGION}" --repository-name "${ORB_STR_REPO}" --image-scanning-configuration scanOnPush=true
-        else
-            aws ecr create-repository --profile "${ORB_STR_PROFILE_NAME}" --region "${ORB_STR_REGION}" --repository-name "${ORB_STR_REPO}" --image-scanning-configuration scanOnPush=false
-        fi
+
+    IMAGE_SCANNING_CONFIGURATION="scanOnPush=true"
+    if [ "$ORB_BOOL_REPO_SCAN_ON_PUSH" != "1" ]; then
```
Comment (EricRibeiro) [inline src/scripts/create_repo.sh:14]:
```suggestion
    if [ "$ORB_BOOL_REPO_SCAN_ON_PUSH" -ne 1 ]; then
```

File: src/scripts/create_repo.sh [hunk 4]
```diff
@@ -2,15 +2,32 @@
 ORB_STR_REGION="$(circleci env subst "${ORB_STR_REGION}")"
 ORB_STR_REPO="$(circleci env subst "${ORB_STR_REPO}")"
 ORB_STR_PROFILE_NAME="$(circleci env subst "${ORB_STR_PROFILE_NAME}")"
+ORB_ENUM_ENCRYPTION_TYPE="$(circleci env subst "${ORB_ENUM_ENCRYPTION_TYPE}")"
```
Comment (EricRibeiro) [inline src/scripts/create_repo.sh:5]: `enum` is a static string. There is no need to expand it.

**Review by EricRibeiro — CHANGES_REQUESTED** [review]

File: src/scripts/create_repo.sh (hunk 3 above)
Comment (brivu) [reply src/scripts/create_repo.sh]: changed in latest commit

**Review by EricRibeiro — APPROVED** [review]

**Pull request merged**